### PR TITLE
Add wiki notice for articles in need of cleanup or rewrite

### PR DESCRIPTION
**Pull request by cl8n**
osu version of https://en.wikipedia.org/wiki/Template:Cleanup_rewrite. `outdated` will override this since it'd be a little redundant

I feel like these messages should be managed on osu-wiki but that's a discussion for some other time :running_man:

### CODE REVIEW

**Comment by LiquidPL** [comment]
To be honest I feel like we should implement at least a rudimentary template system for it.

File: app/Models/Wiki/Page.php [hunk 1]
```diff
@@ -227,6 +227,11 @@ public function isOutdated() : bool
         return $this->page()['header']['outdated'] ?? false;
     }
 
+    public function needsCleanup() : bool
+    {
+        return $this->page()['header']['needs_cleanup'] ?? false;
```
Comment (notbakaneko) [inline app/Models/Wiki/Page.php]:
```suggestion
        return $this->page['header']['needs_cleanup'] ?? false;
```

Comment (cl8n) [reply app/Models/Wiki/Page.php]: `page` is only a function here isn't it

Comment (notbakaneko) [reply app/Models/Wiki/Page.php]: `page()` has been removed, it's `page` now

**Pull request merged**